modules/pel/peltool/peltool.py

```diff
@@ -245,6 +245,22 @@ def parseAndWriteOutput(file: str, output_dir: str, config: Config,
             print(f"No PEL parsed for {file}: {e}")
 
 
+def parseAndSummarise(file: str, config: Config,
```

Comment:
Please add function documentation.

I think you function name is confusing `parseAndSummarise` and `parsePelSummary` because you are just processing only one file, right?

I think you are reading first and parsing so please use name respectively.